app/controllers/fandoms_controller.rb

```diff
@@ -5,32 +5,29 @@ def index
     if @collection
       @media = Media.canonical.by_name - [Media.find_by(name: ArchiveConfig.MEDIA_NO_TAG_NAME)] - [Media.find_by(name: ArchiveConfig.MEDIA_UNCATEGORIZED_NAME)]
       @page_subtitle = @collection.title
-      @medium = Media.find_by_name(params[:medium_id]) if params[:medium_id]
+      @medium = Media.find_by_name(params[:media_id]) if params[:media_id]
       @counts = SearchCounts.fandom_ids_for_collection(@collection)
       @fandoms = (@medium ? @medium.fandoms : Fandom.all).where(id: @counts.keys).by_name
-    elsif params[:medium_id]
-      if @medium = Media.find_by_name(params[:medium_id])
-         @page_subtitle = @medium.name
-        if @medium == Media.uncategorized
-          @fandoms = @medium.fandoms.in_use.by_name
-        else
-          @fandoms = @medium.fandoms.canonical.by_name.with_count
-        end
-      else
-        raise ActiveRecord::RecordNotFound, "Couldn't find media category named '#{params[:medium_id]}'"
-      end
+    elsif params[:media_id]
+      @medium = Media.find_by_name!(params[:media_id]) # rubocop:disable Rails/DynamicFindBy
```

Comment:
Since it's a method we add, you can put it on the list here:
https://github.com/otwcode/otwarchive/blob/fd5046c72ca2b2f6874aaaf839bd54a5410e2677/.rubocop.yml#L134-L137